src/main/java/core/basesyntax/Model/Bulldozer.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax.Model;
+
+import core.basesyntax.Base.Machine;
+
+public class Bulldozer extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Bulldozer started its work");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("Bulldozer stopped working");
```

Comment:
Similar to the `doWork` method, the message in the `stopWork` method should be more informative and indicate the type of machine. For instance: 'Bulldozer truck stopped working.'